src/main/java/movableInterface/MovablePoint.java

```diff
@@ -0,0 +1,77 @@
+package movableInterface;
+
+public class MovablePoint implements Movable {
+    private int x;
+    private int y;
+    private int xSpeed;
+    private int ySpeed;
+
+
+    MovablePoint(int x, int y, int xSpeed, int ySpeed) {
+        this.x = x;
+        this.y = y;
+        this.xSpeed = xSpeed;
+        this.ySpeed = ySpeed;
+    }
+
+    @Override
+    public void moveUp() {
+        int y = this.getY() + this.getySpeed();
+        this.setY(y);
+    }
+
+    @Override
+    public void moveDown() {
+        int y = this.getY() - this.getySpeed();
+        this.setY(y);
+    }
+
+    @Override
+    public void moveLeft() {
+        int x = this.getX() - this.getxSpeed();
+        this.setX(x);
+    }
+
+    @Override
+    public void moveRight() {
+        int x = this.getX() + this.getxSpeed();
+        this.setX(x);
+    }
+
+    @Override
+    public String toString() {
+        return "coordinates: [x: " + x + ", y: " + y + "]";
+    }
+
+    int getX() {
+        return x;
+    }
+
+    int getY() {
+        return y;
+    }
+
+    int getxSpeed() {
```

Comment:
Just a small thing here:
When we name methods we should follow accepted format.
In general it's `camelCase`, so in this exact case method name should be:
`getXSpeed()`

Reply:
This is applicable to all the methods below